From dc7beaec0920c149e7c052898d873d5ab3357033 Mon Sep 17 00:00:00 2001
From: zhang sanshan <b51434@freescale.com>
Date: Tue, 5 Jan 2016 16:58:13 +0800
Subject: [PATCH] MA-7468 Pass cts case
 android.security.cts.EncryptionTest#testConfig

This case will check whether it have driver which named xts-aes-neon or xts-aes-neonbs in /proc/crypto.
Add these two node through enable the macro.
This patch use "make savedefconfig ARCH=arm" to update android configure file.

Signed-off-by: zhang sanshan <b51434@freescale.com>
---
 arch/arm/configs/imx_v7_android_defconfig | 16 +++++-----------
 1 file changed, 5 insertions(+), 11 deletions(-)

diff --git a/arch/arm/configs/imx_v7_android_defconfig b/arch/arm/configs/imx_v7_android_defconfig
index f23b39e..dff9adb 100644
--- a/arch/arm/configs/imx_v7_android_defconfig
+++ b/arch/arm/configs/imx_v7_android_defconfig
@@ -14,9 +14,9 @@ CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_KALLSYMS_ALL=y
 CONFIG_EMBEDDED=y
-CONFIG_PERF_EVENTS=y
 # CONFIG_SLUB_DEBUG is not set
 # CONFIG_COMPAT_BRK is not set
+CONFIG_PROFILING=y
 CONFIG_MODULES=y
 CONFIG_MODULE_UNLOAD=y
 CONFIG_MODVERSIONS=y
@@ -47,7 +47,6 @@ CONFIG_KSM=y
 CONFIG_CMA=y
 CONFIG_ZSMALLOC=y
 CONFIG_SECCOMP=y
-CONFIG_SECCOMP_FILTER=y
 CONFIG_CMDLINE="noinitrd console=ttymxc0,115200"
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
@@ -61,6 +60,7 @@ CONFIG_ARM_IMX7D_CPUFREQ=y
 CONFIG_CPU_IDLE=y
 CONFIG_VFP=y
 CONFIG_NEON=y
+CONFIG_KERNEL_MODE_NEON=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 CONFIG_BINFMT_MISC=y
 CONFIG_PM_AUTOSLEEP=y
@@ -225,6 +225,7 @@ CONFIG_BLK_DEV_RAM_SIZE=8192
 CONFIG_SENSORS_FXOS8700=y
 CONFIG_SENSORS_FXAS2100X=y
 CONFIG_UID_STAT=y
+CONFIG_UID_CPUTIME=y
 CONFIG_EEPROM_AT24=y
 CONFIG_EEPROM_AT25=y
 # CONFIG_SCSI_PROC_FS is not set
@@ -287,7 +288,6 @@ CONFIG_RTL8821AS=m
 CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_KEYRESET=y
 CONFIG_KEYBOARD_GPIO=y
-CONFIG_KEYBOARD_SNVS_PWRKEY=y
 CONFIG_KEYBOARD_IMX=y
 # CONFIG_INPUT_MOUSE is not set
 CONFIG_INPUT_JOYSTICK=y
@@ -518,7 +518,6 @@ CONFIG_MXC_GPU_VIV=y
 CONFIG_GPU_LOW_MEMORY_KILLER=y
 CONFIG_MXC_IPU_V3_PRE=y
 CONFIG_MXC_MIPI_CSI2=y
-CONFIG_MXC_MLB150=y
 CONFIG_MXC_SIM=y
 CONFIG_LEDS_GPIO=y
 CONFIG_LEDS_TRIGGERS=y
@@ -598,15 +597,12 @@ CONFIG_NLS_ASCII=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_NLS_ISO8859_15=y
 CONFIG_NLS_UTF8=y
-CONFIG_DEBUG_FS=y
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_PANIC_TIMEOUT=5
-CONFIG_SCHED_DEBUG=y
 CONFIG_DETECT_HUNG_TASK=y
+CONFIG_PANIC_TIMEOUT=5
 CONFIG_SCHEDSTATS=y
 CONFIG_TIMER_STATS=y
 # CONFIG_DEBUG_BUGVERBOSE is not set
-CONFIG_FTRACE=y
 CONFIG_ENABLE_DEFAULT_TRACERS=y
 CONFIG_CORESIGHT=y
 CONFIG_CORESIGHT_LINK_AND_SINK_TMC=y
@@ -618,7 +614,6 @@ CONFIG_SECURITY_SELINUX=y
 CONFIG_SECURITY_APPARMOR=y
 CONFIG_CRYPTO_USER=y
 CONFIG_CRYPTO_TEST=m
-CONFIG_CRYPTO_CCM=y
 CONFIG_CRYPTO_GCM=y
 CONFIG_CRYPTO_CTS=y
 CONFIG_CRYPTO_LRW=y
@@ -632,6 +627,7 @@ CONFIG_CRYPTO_RMD320=y
 CONFIG_CRYPTO_SHA512=y
 CONFIG_CRYPTO_TGR192=y
 CONFIG_CRYPTO_WP512=y
+CONFIG_CRYPTO_AES_ARM_BS=y
 CONFIG_CRYPTO_BLOWFISH=y
 CONFIG_CRYPTO_CAMELLIA=y
 CONFIG_CRYPTO_TWOFISH=y
@@ -645,5 +641,3 @@ CONFIG_CRC7=y
 CONFIG_FONTS=y
 CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
-CONFIG_PROFILING=y
-CONFIG_UID_CPUTIME=y
-- 
1.8.0

